### PR TITLE
[BCTC-253] - need new type for homework submission

### PR DESCRIPTION
**Pull request by leocabbage2008**

### CODE REVIEW

File: fs/index.ts [hunk 1]
```diff
@@ -75,3 +75,12 @@ export interface IAnnouncements {
   currentPage: number;
   totalAnnouncements: number;
 }
+
+export interface IHomework {
```
Comment (chuxu200328) [inline fs/index.ts]: ISubmission is better

File: fs/index.ts [hunk 2]
```diff
@@ -75,3 +75,12 @@ export interface IAnnouncements {
...
+  attachment?: FileList;
+  content?: string;
+  studentId: string;
+  classId: string;
+  classIndex: string;
+  comment:string;
```
Comment (chuxu200328) [inline fs/index.ts]: what is comment?

**Review by chuxu200328 — CHANGES_REQUESTED** [review]

File: fs/index.ts (hunk 2 above)
Comment (leocabbage2008) [reply fs/index.ts]: this, 
image

Comment (chuxu200328) [reply fs/index.ts]: that should be the original content. the new filed should be link.

**Review by chuxu200328 — APPROVED** [review]

**Pull request merged**